jobs/ci-kubernetes-node-kubelet-conformance.sh

```diff
@@ -19,4 +19,23 @@ set -o pipefail
 set -o xtrace
 
 # PROJECT="k8s-jkns-ci-node-e2e"
-test/e2e_node/jenkins/conformance/conformance-jenkins.sh test/e2e_node/jenkins/conformance/jenkins-conformance.properties
```

Comment:
at one point we mapped in the docker client binary (`/usr/bin/docker`) but this doesn't work anymore because it's not statically linked.

the new recommended way (as best I understand) is to install the docker client inside the image and map the docker socket in, which is [what we do for the unit tests right now](https://github.com/kubernetes/test-infra/blob/master/jenkins/gotest-dockerized.sh#L50-L51). Since the node kubekins image is based off kubekins test, we should have docker installed, so mapping in the docker socket and setting privileged mode should be sufficient.

that said, there are more gotchas: if you call docker from within docker and try to use volume mounts, the paths are based on the *host* path, not the path in the container you're calling from. this is why we use `REPO_DIR` and `HOST_ARTIFACTS_DIR` in `gotest-dockerized.sh`. You'll probably need to look at `conformance-jenkins.sh` to see if something similar is needed here.